src/stream/src/task/stream_manager.rs

```diff
@@ -647,13 +648,17 @@ impl LocalBarrierWorker {
                     None => actor.right_future(),
                 };
                 let instrumented = monitor.instrument(traced);
+                let with_config =
+                    crate::CONFIG.scope(self.actor_manager.env.config().clone(), instrumented);
+
+                // TODO: are the following lines still useful?
```

Comment:
I think we can remove them with all stuff releated to `task_local_alloc` since it has been deprecated for a while.

Reply:
Yes, let's do it in another PR.